LoggerFinal/src/main/java/com/Logger/DemoApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.Logger;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class DemoApplication {
```

Comment:
Change name of class from DemoApplication to something more meaningful for this project